grammars/javascript.cson

```diff
@@ -555,6 +571,13 @@
   }
 ]
 'repository':
+  'numbers':
+    'patterns': [
+      {
+        'match': '\\b((0(x|X)[0-9a-fA-F]+)|(0(b|B)[01]+)|(0(o|O)[0-7]+)|([0-9]+(\\.[0-9]+)?)((e|E)[+-]?[0-9]+)?)\\b'
```

Comment:
Can JavaScript decimal literals have an optional sign on the front of them too? Or is it like in Python where there are only positive literals that get negated?